spec/lib/data_migration_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'rails_helper'
+require 'data_migration'
+
+RSpec.describe DataMigration do
+  let!(:unmigrated_asset) { FactoryGirl.create(:asset, uuid: 'some-uuid') }
+  let!(:migrated_asset) { FactoryGirl.create(:asset) }
```

Comment:
Won't this asset have a `#uuid` given that there's a default block in the `field :uuid` definition? Even if that's not true, I think it might be clearer to explicitly set the `uuid` to `nil` in this set-up.

Reply:
That would be clearer - addressed in 7dfc266